mpas_analysis/shared/mpas_xarray/mpas_xarray.py

```diff
@@ -543,4 +532,59 @@ def _parse_dataset_time(ds, inTimeVariableName, calendar,
     return dsOut  # }}}
 
 
+def process_chunking(ds, chunking):  # {{{
+    """
+    Computes chunking for a dataset.
+
+    Parameters
+    ----------
+    ds : input xarray Dataset.
+        Dataset to be chunked.
+
+    chunking : None, True, dict, optional
+        If present, applies specialized chunking rule, otherwise if None do not
```

Comment:
Below, you support chunking as an integer but you don't seem to support `chunking == True`.